Exercise_B/index.html

```diff
@@ -7,28 +7,90 @@
     <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
     <!-- Link and CDN required for bootstrap -->
-
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+      crossorigin="anonymous"
+    />
+    <!-- Custom Stylesheet -->
+    <style>
+      .bg-success {
+        background-color: rgb(123, 226, 158) !important;
+      }
+    </style>
+    
     <!-- Google Font - Lato -->
     <link
       href="https://fonts.googleapis.com/css?family=Lato"
       rel="stylesheet"
     />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
 
   <body>

```

Comment:
Well done for implementing this bootstrapped version of the flowershop website 

Fantastic use of the bootstrap column system to facilitate the icons appearing in the correct locations.
Superb use of bootstrap utilities to reduce the amount of CSS required. 

Consider adding no gutters or removing the container so your website can take up the full browser space. unless of course you meant to do that!